packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -28,6 +28,12 @@ export interface OdspConnectionConfig {
 	 * RaaS Drive Id of the tenant where Fluid containers are created
 	 */
 	driveId: string;
+
+	/**
+	 * Specifies the file path where Fluid files are created.
+	 * If not specified, Fluid files will be created at the root of the drive.
+	 */
+	filePath?: string;
```

Comment:
I'd suggest making this non-optional (at least for now) since it would match with the underlying `createOdspCreateContainerRequest` and it's easier to relax it later than to make it stricter.